### DISCUSSION TITLE
Issue After Upgrading Netbox from 2.9.4 to 3.6.5.

### DISCUSSION THREAD

**Discussion by petrospetrou1995** [netbox-community/netbox#14264] [Help Wanted!]
### NetBox version

v2.9.4

### Python version

3.8

### Steps to Reproduce

Using Clone from the Git Repository Netbox installation.

Using command sudo ./upgrade.shthe 

### Expected Behavior

 Static Media Failure
The following static media file failed to load: [netbox-print.css](https://netbox.myFQDN.here/netbox/static/netbox-print.css)

Check the following:

    manage.py collectstatic was run during the most recent upgrade. This installs the most recent iteration of each static file into the static root path.
    The HTTP service (e.g. nginx or Apache) is configured to serve files from the STATIC_ROOT path. Refer to [the installation documentation](https://netbox.readthedocs.io/en/stable/installation/) for further guidance.
        STATIC_URL: /netbox/static/
    The file netbox-print.css exists in the static root directory and is readable by the HTTP process.

Click [here](https://netbox.myFQDN.here/netbox/) to attempt loading NetBox again. 
nginx.conf

### Observed Behavior

Page not loading

**Comment by petrospetrou1995** [comment]
Hello I am running manually incremental upgrade of netbox, so far i could upgrade it to version v3.1.11. Therefor from 3.2.0 and above am getting the following error.
![f4bc0eb6-d4da-43f3-9f18-a65b33985bf5](https://github.com/netbox-community/netbox/assets/51456874/20f6bedf-f702-4dd7-ab02-98d4bee0b3c0)
![6718b9b8-0403-4573-a7b1-42702a307ea6](https://github.com/netbox-community/netbox/assets/51456874/9daafa40-ead3-4dbd-bbcc-97819b4b89ba)

**Reply by candlerb** [reply]
I can't read your fuzzy screenshot very well but I think it says "found N sites with legacy contact data". This is preventing your migration from completing.

Best solution is to edit those sites and remove the legacy contact data (migrating it to Contact objects if you want to keep it).

If you're happy just to lose the contact data, then do what it suggests:

```
NETBOX_DELETE_LEGACY_DATA=1 ./upgrade.sh
```
